koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -657,48 +662,53 @@ def populate_volume_label_summary_table(self):
         )
         self._commit_and_vacuum(table_name, agg_sql, bind_params=list(agg_sql_params))
 
-    def populate_markup_cost(self, markup, cluster_id=None):
-        """Set markup costs in the database."""
-        with schema_context(self.schema):
-            if cluster_id:
-                CostSummary.objects.filter(cluster_id=cluster_id).update(
-                    markup_cost=((F('pod_charge_cpu_core_hours')
-                                  + F('pod_charge_memory_gigabyte_hours')
-                                  + F('persistentvolumeclaim_charge_gb_month')
-                                  + F('infra_cost')) * markup)
-                )
-                CostSummary.objects.filter(cluster_id=cluster_id).update(
-                    project_markup_cost=((F('pod_charge_cpu_core_hours')
-                                          + F('pod_charge_memory_gigabyte_hours')
-                                          + F('persistentvolumeclaim_charge_gb_month')
-                                          + F('project_infra_cost')) * markup)
-                )
-            else:
-                CostSummary.objects.update(
-                    markup_cost=((F('pod_charge_cpu_core_hours')
-                                  + F('pod_charge_memory_gigabyte_hours')
-                                  + F('persistentvolumeclaim_charge_gb_month')
-                                  + F('infra_cost')) * markup)
-                )
-                CostSummary.objects.update(
-                    project_markup_cost=((F('pod_charge_cpu_core_hours')
-                                          + F('pod_charge_memory_gigabyte_hours')
-                                          + F('persistentvolumeclaim_charge_gb_month')
-                                          + F('project_infra_cost')) * markup)
-                )
-
-    def populate_ocp_on_aws_markup_cost(self, aws_markup, ocp_markup, cluster_id):
+    def populate_markup_cost(self, aws_markup, ocp_markup, cluster_id):
         """Set markup cost for OCP-on-AWS in Cost Summary."""
```

Comment:
Does this need to stay `OCP-on-AWS`? The function seems generic enough that it looks like we can change it to `OCP-on-Cloud` and `aws_markup => cloud_markup`